Feature/create flask api layout (#1)
makeopensource/MakeOfficeHours: README.md

# MakeOfficeHours

[![linting: pylint](https://img.shields.io/badge/linting-pylint-yellowgreen)](https://github.com/pylint-dev/pylint)

# api

> A simple flask api server that handels students enqueue and dequeue

Reviewer: ```suggestion
A simple flask api server that handles students enqueue and dequeue
```

Reviewer 2: This is describing the main (backend) service in the core readme; it's what people will first see when learning about this project, so it should be informative, correct, and not belittling of the project.

```suggestion
A Flask API server that handles enqueuing and dequeuing students from the office hours queue.
```

(Or similar, since I don't know the full scope of this services' responsibilies.)

> ### how to run

Reviewer 2: Consistent header capitalization

```suggestion
### How to run
```

#### Debug
```bash
flask --app api run --debug
```

> ### pylint

Reviewer 2: ```suggestion
### Pylint
```

> Project uses pylint to keep the code style concurrent

Reviewer 2: Pylint is a proper noun; capitalize it in English writing (excluding technical usages, such as in commands).

```suggestion
Project uses Pylint to keep the code style concurrent
```

<!-- TODO: ask if we should use google's pylint setting -->

You can run the pylint on the api folder by doing the following
```bash
pylint api/
```

#### Formatter

> Currently using the black formatter https://github.com/psf/black

Reviewer 2: Black is also a proper noun. "Currently" makes it sound like we're planning to switch, so let's drop it.

```suggestion
Using the Black formatter https://github.com/psf/black
```
